Cargo.toml

```diff
@@ -127,6 +127,9 @@ optional = true
 [dev-dependencies.rusty-hook]
 version = "0.11.2"
 
+[dev-dependencies.serial_test]
+version = "0.9.0"
```

Comment:
instead of running the tests one-by-one, which is a workaround, we shouldn't be using `&'static` for storage in the first place; this will require snarkVM-side changes first